.circleci/config.yml

```diff
@@ -530,7 +530,7 @@ jobs:
                   fi
             - *store_artifacts
       - *persist_to_workspace
-  Server 4_1:
+  Server 4_5:
```

Comment:
I think it will be better to revert the changes in this file and just remove the 4_1 job and create a fresh one.
each job has it's own settings, for example:
`Server 6_0` should have the upload packs to production steps.
`Server Master` should have the option of running build without AMI (special builds).